src/components/elements/side-tab.js

```diff
@@ -36,15 +48,35 @@ const StyledTab = styled.li`
         }
     }
 `
+const StyledDropDown = styled.li`
+    padding: 1rem 0;
+    color: var(--color-red);
+    border-bottom: 1px solid var(--color-red);
+    display: flex;
+    justify-content: space-between;
 
-const Tab = ({ active_tab, label, onClick, text }) => {
+    p {
+        font-size: var(--text-size-s);
+        color: var(--color-red);
+        font-size: 2rem;
+    }
+`
+const ArrowWrapper = styled(Arrow)`
+    transform: ${props =>
+        props.what_is_active_tab === '-' ? 'rotate(0deg)' : 'rotate(180deg)'};
+`
+const Tab = ({ active_tab, label, onClick, text, mobile }) => {
     const className = active_tab === label ? 'tab-active' : ''
```

Comment:
and yes, i guess, we can make this into class component instead as you suggested